### DISCUSSION TITLE
Licensing Models: Per-x Variations

### DISCUSSION THREAD

**Discussion by JeremyVyska** [Spare-Brained-Community/Spare-Brained-Licensing#1] [Ideas]
I could use some feedback on what folks _need_ from this functionality.

My first use case was per-environment licensing only.  So, the other possible needs that I could think of:

- Per User
- Per Company
- Per Transactions (of whatever?) per some Period of time.

I'd like to know what people might want in that space before trying to architect it.

**Comment by omerfaruknav** [comment]
Hi, We currently need per company licensing, and searching for a way to do that. Did you  do development for this way of licensing?
